pybrake/flask.py

```diff
@@ -28,6 +28,41 @@
 _UNKNOWN_ROUTE = "UNKNOWN"
 
 
+def request_filter(notice):
+    ctx = notice["context"]
+    ctx["method"] = request.method
```

Comment:
And what will happen if this is called when request is not available?

Reply:
Added check if the request object is not available.